tests/sanity/sanity.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+VENV=/tmp/ansible-test-venv
+ANSIBLE_COLLECTION=freeipa-ansible_freeipa
+
+virtualenv "$VENV"
+# shellcheck disable=SC1091
+source "$VENV"/bin/activate
+
+python -m pip install --upgrade pip
+pip install galaxy_importer
+
+rm -f "$ANSIBLE_COLLECTION"-*.tar.gz
+rm -f importer_result.json
+
+utils/build-galaxy-release.sh
+
+export GALAXY_IMPORTER_CONFIG=tests/sanity/galaxy-importer.cfg
+
+collection=$(ls -1 "$ANSIBLE_COLLECTION"-*.tar.gz)
+echo "Running: python -m galaxy_importer.main $collection"
+
+python -m galaxy_importer.main "$collection"
+
+rm -rf "$VENV"
```

Comment:
Maybe changing this to...
```sh
importer_output=$(mktemp)
python -m galaxy_importer.main "$collection" | tee "${importer_output}"

rm -rf "$VENV"

{ cat "${importer_output}" ; rm "${importer_output}" } | grep "^ERROR" >/dev/null
```

... and we get an error.

Although I don't like the `grep` test, as I don't know if it will get all errors.

Reply:
I did it a bit different to also print the error lines in red.